### PR TITLE
Enable Ecal DQM in Phase 2 workflows

### PR DESCRIPTION
**Pull request by tanmaymudholkar**
#### PR description:

This PR enables the ECAL DQM validation (simhits, digis, rechits, clusters, DQM) In the new phase2 workflows. This validation was so far not run for the phase2 workflows.

With this PR, the DQMIO output should fill the plots needed in the DataLayouts/Ecal subdirectory.

#### PR validation:

We have run the basic battery of tests:

`runTheMatrix.py -l limited -i all --ibeos
`

#### if this PR is a backport please specify the original PR:

This PR is not a backport. Corresponding PR to 10_6_X: https://github.com/cms-sw/cmssw/pull/27483

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 1]
```diff
@@ -103,3 +103,46 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
 
+from SimGeneral.MixingModule.mixObjects_cfi import theMixObjects
+from SimGeneral.MixingModule.mixPoolSource_cfi import *
+from SimGeneral.MixingModule.digitizers_cfi import *
+
+mixNoPileUp = cms.EDProducer("MixingModule",
```
Comment (makortel) [inline Configuration/StandardSequences/python/Validation_cff.py]: Why is additional instance of `MixingModule` needed? IIRC `MixingModule` is already run in playback mode in jobs with VALIDATION step. I also don't see `mixNoPileUp` product label used in this PR nor in `master`.

Comment (tanmaymudholkar) [reply Configuration/StandardSequences/python/Validation_cff.py]: @makortel : Thanks for the correction. Indeed we don't need an additional instance of the mixing module. The test workflows succeed with it removed. I have modified the PR.

Comment (tanmaymudholkar) [reply Configuration/StandardSequences/python/Validation_cff.py]: @jfernan2 I believe the tests need to be restarted? Thanks!

**Comment by jfernan2** [comment]
please test workflow 20034.0

**Comment by jfernan2** [comment]
Same comment as in the backport:
@tanmaymudholkar  the PR is not only adding 1911 plots related to ECAL in the Phase2 relvals but also 65 plots on EcalDigisV/SelectiveReadout for every MC workflow tested:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_11_0_X_2019-07-17-2300+81ef0a/32699/dqm-histo-comparison-summary.html
Besides, in the case of the Phase2 workflows, the FED folder is also added, which I believe it makes sense only for DQM Online
Is all this expected?

**Comment by tanmaymudholkar** [comment]
Discussion of this issue on the backport comment [here](https://github.com/cms-sw/cmssw/pull/27483#issuecomment-513282999).

**Comment by jfernan2** [comment]
+1
Thanks @tanmaymudholkar It is clear now.
Please try to fix the other issue I sent you about warnings from ECAL in logs from Cosmics and other workflows

**Comment by civanch** [comment]
+1

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 2]
```diff
@@ -103,3 +103,23 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
 
+from Validation.EcalClusters.ecalClustersValidationSequence_cff import ecalClustersValidationSequence
```
Comment (fabiocos) [inline Configuration/StandardSequences/python/Validation_cff.py]: @tanmaymudholkar @jfernan2 I repeat here my comment on the 10_6_X backport https://github.com/cms-sw/cmssw/pull/27483#pullrequestreview-265681038 : this does not seem to me the best place where to insert this definition, that should better go in the ECAL validation fragment.

**Comment by fabiocos** [comment]
hold

**Comment by jfernan2** [comment]
-1
Until python configuration is moved to proper file

File: Configuration/StandardSequences/python/Validation_cff.py (hunk 2 above)
Comment (tanmaymudholkar) [reply Configuration/StandardSequences/python/Validation_cff.py]: @fabiocos @jfernan2 Indeed. Sequence definition moved to better location in the latest commit.

**Comment by jfernan2** [comment]
please test

**Comment by jfernan2** [comment]
+1

**Comment by fabiocos** [comment]
unhold 

@tanmaymudholkar thank you for addressing my comment
@civanch I would say it is ok, could you please check and sign it again?

**Comment by civanch** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Pull request merged**